### PR TITLE
[Done] Translation of tutorial/nav.yml ..

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

File: content/tutorial/nav.yml [hunk 1]
```diff
@@ -1,111 +1,111 @@
-- title: Tutorial
+- title: الدليل التطبيقي
```
Comment (Aissaoui-Ahmed) [inline content/tutorial/nav.yml]: title: الدليل التطبيقي >  title: الدورة التعليمية

File: content/tutorial/nav.yml [hunk 2]
```diff
@@ -1,111 +1,111 @@
...
   items:
     - id: before-we-start-the-tutorial
-      title: Before We Start the Tutorial
+      title: 	قبل أن نبدأ بالدليل التطبيقي
```
Comment (Aissaoui-Ahmed) [inline content/tutorial/nav.yml]: هنا أيضا

File: content/tutorial/nav.yml [hunk 3]
```diff
@@ -1,111 +1,111 @@
...
       href: /tutorial/tutorial.html#before-we-start-the-tutorial
       forceInternal: true
       subitems:
         - id: what-are-we-building
-          title: What Are We Building?
+          title: 	ماذا سنبني؟
```
Comment (Aissaoui-Ahmed) [inline content/tutorial/nav.yml]: remove "TAB"

File: content/tutorial/nav.yml [hunk 4]
```diff
@@ -1,111 +1,111 @@
...
           href: /tutorial/tutorial.html#what-are-we-building
           forceInternal: true
         - id: prerequisites
-          title: Prerequisites
+          title: 	المتطلّبات الأساسيّة
           href: /tutorial/tutorial.html#prerequisites
           forceInternal: true
     - id: setup-for-the-tutorial
-      title: Setup for the Tutorial
+      title: الإعداد من أجل الدليل
```
Comment (Aissaoui-Ahmed) [inline content/tutorial/nav.yml]: هنا أيضا

File: content/tutorial/nav.yml [hunk 5]
```diff
@@ -1,111 +1,111 @@
...
       href: /tutorial/tutorial.html#setup-for-the-tutorial
       forceInternal: true
       subitems:
         - id: setup-option-1-write-code-in-the-browser
-          title: "Option 1: Write Code in the Browser"
+          title: "الخيار الأول للإعداد: كتابة الشيفرة في المتصفح"
```
Comment (Aissaoui-Ahmed) [inline content/tutorial/nav.yml]: الخِيار 1: كتابة الشيفرة في المتصفح > الخيار الأول للإعداد: كتابة الشيفرة في المتصفح

File: content/tutorial/nav.yml [hunk 6]
```diff
@@ -1,111 +1,111 @@
...
           href: /tutorial/tutorial.html#function-components
           forceInternal: true
         - id: taking-turns
-          title: Taking Turns
+          title: أخذ الأدوار
           href: /tutorial/tutorial.html#taking-turns
           forceInternal: true
         - id: declaring-a-winner
-          title: Declaring a Winner
+          title: 	التصريح عن الفائز
```
Comment (Aissaoui-Ahmed) [inline content/tutorial/nav.yml]: "TAB"

File: content/tutorial/nav.yml [hunk 7]
```diff
@@ -1,111 +1,111 @@
...
       forceInternal: true
       subitems:
         - id: setup-option-1-write-code-in-the-browser
-          title: "Option 1: Write Code in the Browser"
+          title: "الخيار الأول للإعداد: كتابة الشيفرة في المتصفح"
           href: /tutorial/tutorial.html#setup-option-1-write-code-in-the-browser
           forceInternal: true
         - id: setup-option-2-local-development-environment
-          title: "Option 2: Local Development Environment"
+          title: "الخيار الثاني للإعداد: بيئة التطوير المحليّة"
```
Comment (Aissaoui-Ahmed) [inline content/tutorial/nav.yml]: الخِيار 2 > الخيار الثاني للإعداد

**Comment by 3imed-jaberi** [comment]
Hi @Aissaoui-Ahmed , 
Thanks for your suggestions ❤️  .. 
Please add this translation and associated terms to the glossary issue #122 for discussion ...

**Comment by celyes** [comment]
@3imed-jaberi I can review any upcoming pull requests, add me if you want.

File: content/tutorial/nav.yml [hunk 8]
```diff
@@ -1,111 +1,111 @@
...
       forceInternal: true
       subitems:
         - id: what-is-react
-          title: What Is React?
+          title: ما هي React؟
           href: /tutorial/tutorial.html#what-is-react
           forceInternal: true
         - id: inspecting-the-starter-code
-          title: Inspecting the Starter Code
+          title: تفحّص شيفرة البدء
```
Comment (mohshbool) [inline content/tutorial/nav.yml]:
```suggestion
          title: تَفَحُّص شيفرة البدء
```

File: content/tutorial/nav.yml [hunk 9]
```diff
@@ -1,111 +1,111 @@
...
       forceInternal: true
       subitems:
         - id: lifting-state-up
-          title: Lifting State Up
+          title: رفع الحالة للمستوى الأعلى
           href: /tutorial/tutorial.html#lifting-state-up
           forceInternal: true
         - id: why-immutability-is-important
-          title: Why Immutability Is Important
+          title: لماذا تكون عدم القابلية للتغير مهمة؟
```
Comment (mohshbool) [inline content/tutorial/nav.yml]:
```suggestion
          title: السبب وراء أهمية عدم قابلية التغيير
```

File: content/tutorial/nav.yml [hunk 10]
```diff
@@ -1,111 +1,111 @@
...
           href: /tutorial/tutorial.html#function-components
           forceInternal: true
         - id: taking-turns
-          title: Taking Turns
+          title: أخذ الأدوار
           href: /tutorial/tutorial.html#taking-turns
           forceInternal: true
         - id: declaring-a-winner
-          title: Declaring a Winner
+          title: التصريح عن الفائز
```
Comment (mohshbool) [inline content/tutorial/nav.yml]:
```suggestion
          title: إعلان فائز
```

File: content/tutorial/nav.yml (hunk 9 above)
Comment (3imed-jaberi) [reply content/tutorial/nav.yml]: What do you think of this expression `لماذا عدم قابلية التغير مهمة؟` 🤔 .. I think it's better ✔️ ...

Comment (mohshbool) [reply content/tutorial/nav.yml]: I think it's a bit weak imo, you don't have to translate word by word, I asked some people and they agree with my suggestion according to grammar rules

**Comment by mohshbool** [comment]
🎉

**Pull request merged**